### ISSUE TITLE
Setup continuous integration

### ISSUE THREAD

**Issue by graeme-a-stewart** [AIDASoft/podio#42]
We should setup some continuous integration for this repo.

This would involve preparing a few docker containers with a self-contained environment on the platforms we want to test and using Travis to clone, build and test the code.

The https://github.com/HSF/prmon repo shows how to do that in a pretty straight forward way.

**Comment by gaede** [comment]
Absolutely. We might base it on things we are doing for the LC software, incl. DD4hep. This might be less additional effort. I will investigate this...
